src/lightning/fabric/fabric.py

```diff
@@ -347,7 +347,7 @@ def setup_dataloaders(
     def _setup_dataloader(
         self, dataloader: DataLoader, use_distributed_sampler: bool = True, move_to_device: bool = True
     ) -> DataLoader:
-        """Set up a single dataloader for accelerated training.
+        r"""Set up a single dataloader for accelerated training.
```

Comment:
why is this added everywhere? Seems highly unnecessary?

Reply:
was not sure how the RST syntax works outside strings which are not marked as to be RST